app/services/DepartureService.scala

```diff
@@ -31,21 +31,23 @@ import scala.concurrent.ExecutionContext
 import scala.concurrent.Future
 import scala.xml.NodeSeq
 
-class DepartureService @Inject()(departureIdRepository: DepartureIdRepository)(implicit clock: Clock, ec: ExecutionContext) {
+class DepartureService @Inject() (departureIdRepository: DepartureIdRepository)(implicit clock: Clock, ec: ExecutionContext) {
   import XMLTransformer._
   import XmlMessageParser._
 
-  def makeMessageWithStatus(departureId: DepartureId,
-                            messageId: MessageId,
-                            messageCorrelationId: Int,
-                            messageType: MessageType): ReaderT[ParseHandler, NodeSeq, MessageWithStatus] =
+  def makeMessageWithStatus(
+    departureId: DepartureId,
+    messageId: MessageId,
+    messageCorrelationId: Int,
+    messageType: MessageType
+  ): ReaderT[ParseHandler, NodeSeq, MessageWithStatus] =
     for {
       _          <- correctRootNodeR(messageType)
       dateTime   <- dateTimeOfPrepR
       xmlMessage <- updateMesSenMES3(departureId, messageCorrelationId)
     } yield MessageWithStatus(messageId, dateTime, messageType, xmlMessage, SubmissionPending, messageCorrelationId)
 
-  def createDeparture(enrolmentId: Ior[TURN, EORINumber], nodeSeq: NodeSeq, channelType: ChannelType, boxOpt: Option[Box]): Future[ParseHandler[Departure]] =
+  def createDeparture(customerId: String, nodeSeq: NodeSeq, channelType: ChannelType, boxOpt: Option[Box]): Future[ParseHandler[Departure]] =
```

Comment:
I think I'd prefer to see this as `EnrolmentId` and just grab the customer ID below, just for compile time checking more than anything.